.github/release-to-wiki.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+GIT_REPOSITORY_URL="https://${GITHUB_PERSONAL_ACCESS_TOKEN}@github.com/$GITHUB_REPOSITORY.wiki.git"
+
+tmp_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+(
+    cd "$tmp_dir" || exit 1
+    git init
+    git config user.name "$GITHUB_ACTOR"
+    git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+    git pull "$GIT_REPOSITORY_URL"
+)
+
+mkdir -p "$tmp_dir/$1"
+for file in $(find $1 -maxdepth 1 -type f -name '*' -execdir basename '{}' ';'); do
+    cp "$1/$file" "$tmp_dir/$1"
+done
```

Comment:
Why is this necessary? Why is it only copying top-level files?

Reply:
it really only needs to copy the two zip files, I'll just make it two explicit copies to make it clearer and less generic.

Reply:
Ah, in that case you could just use a simple glob: `cp dir/*.zip dest-dir` 👍

Reply:
Nice! Updated.